scene/gui/text_edit.cpp

```diff
@@ -2851,7 +2851,7 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 				}
 				select_all();
 #else
-				if (k->get_alt()) {
+				if (k->get_alt() || (!k->get_shift() && !k->get_command() && !k->get_control())) {
```

Comment:
This code was added in #18003 (cc @sherjilozair), but I'm not sure of the purpose of this `if (k->get_alt())` check.
The new logic may also not be consistent with the `#ifndef` case just above.